### PR TITLE
[Tizen] Bump Tizen.UIExtensions from 0.6.0-pre1 to 0.8.0-pre1

### PR DESCRIPTION
**Pull request by JoonghyunCho**
### Description of Change

- Bump [Tizen.UIExtensions](https://github.com/Samsung/Tizen.UIExtensions) from 0.6.0-pre1 to 0.8.0-pre1.
- Bump [Samsung Tizen SDK](https://github.com/Samsung/Tizen.NET) from 7.0.100-preview.13.30 to 7.0.300-rc.3.0

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -50,7 +50,7 @@
     <MicrosoftDotNetXHarnessTestRunnersCommonVersion>1.0.0-prerelease.22219.1</MicrosoftDotNetXHarnessTestRunnersCommonVersion>
     <MicrosoftDotNetXHarnessTestRunnersXunitVersion>1.0.0-prerelease.22219.1</MicrosoftDotNetXHarnessTestRunnersXunitVersion>
     <MicrosoftDotNetXHarnessCLIVersion>1.0.0-prerelease.22219.1</MicrosoftDotNetXHarnessCLIVersion>
-    <TizenUIExtensionsVersion>0.6.0-pre1</TizenUIExtensionsVersion>
+    <TizenUIExtensionsVersion>0.8.0-pre1</TizenUIExtensionsVersion>
```
Comment (rookiejava) [inline eng/Versions.props:53]: Shouldn't  `<SamsungTizenSdkPackageVersion>` also be updated to the latest version above?

**Comment by rookiejava** [comment]
Seems you need to rebase

File: eng/Versions.props (hunk 1 above)
Comment (JoonghyunCho) [reply eng/Versions.props:53]: yes yes, I will also update `<SamsungTizenSdkPackageVersion>`. Thanks!

**Review by rookiejava — APPROVED** [review]

**Review by mattleibow — APPROVED** [review]

**Pull request merged**